cni/azure-windows-baremetal.conflist

```diff
@@ -25,6 +25,9 @@
                     "svc.cluster.local"
                 ]
             },
+            windowsSettings: {
+                "disableLoopbackDSR": true
```

Comment:
Since its enabled by default in AKS going forward, thought of having this enable this by default in CNI too.. thats the reason i have this as disableDSR

Reply:
This may impact aks-engine clusters since WinDSR is not enabled in aks-engine clusters. cc @jsturtevant @marosset .

Reply:
@AbelHu i can leave disable as default and AKS can add enableDSR in conflist as required. Im fine with that too.. let me know what you guys think

Reply:
Since its enabled by default in AKS going forward, thought of having this enabled by default in CNI too.. thats the reason i have this as disableDSR
--> @tamilmani1989 IMO CNI options / conflist flags should be generic and not influenced by AKS. As you pointed out in the second comment, they can update conflist accordingly.